.vscode/settings.json

```diff
@@ -17,5 +17,6 @@
     "ssandra",
     "ssandra's",
     "templating"
-  ]
+  ],
```

Comment:
Shouldn't we leave IDE-specific files out of version control?

Reply:
We should :)